File: backend/domain/dto/DiagnosisDTO.java
package com.example._th_project.domain.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DiagnosisDTO {
    private Long diagnosisId;
    private String petName;
    private byte[] diagnosisImg;
    private String disease;
    private String stage;
    private Double riskScore;

    public DiagnosisDTO(Long diagnosisId, String petName, byte[] diagnosisImg, String disease, String stage, Double riskScore){
        this.diagnosisId = diagnosisId;
        this.petName = petName;
        this.diagnosisImg = diagnosisImg;
        this.disease = disease;
        this.stage = stage;
        this.riskScore = riskScore;
    }
}

File: frontend/android/model/DiagnosisDTO.kt
package com.project.meongnyangcare.model

data class DiagnosisDTO(
    val diagnosisId: Long,
    val petName: String,
    val diagnosisImg: String, // 서버에서 Base64 문자열로 받을 경우 String으로 변경 가능
    val disease: String,
    val stage: String,
    val riskScore: Double
)